docs/formatV3.txt

```diff
@@ -591,6 +598,26 @@ These TOTP parameters fields have the following format:
                       default is 0. The format is that of a 40 bit (5 byte)
                       Password Safe time_t value as specified in Section 3.1.3.
 
+[30] The attachment parameters fields are listed below. If an entry contains
+an attachment the fields Att MediaType and Att Content must be present,
+the others are optional (default to empty/zero).
+
+    Att Title = User-friendly name for the attachment
+
+    Att MediaType = MIME type - text string describing Media Type, per
+      http://www.iana.org/assignments/media-types
+
+    Att FileName = Original file name - kept mainly for the extension, to support
+      applications that require this for proper viewing.
+
+    Att FileCreationTime, Att FileModificationTime, Att FileAccessTime =
+      Times associated with the attachment file at the time of attachment.
+      Application may try to restore these when exporting the attachment.
+      Representation is as described in Section 3.1.3.
+
+    Att Content = The binary representation of the attachment.
```

Comment:
I would add something like "interpretation of this field's value is per the media type specified in Att MediaType (0x26)".

Reply:
Added. I might have some time tomorrow to experiment a bit how to fit this into the existing ItemAtt api.
If that works, then that will require minimal to no changes to apps currently supporting attachments.